Motion_planning/resource.md

```diff
@@ -0,0 +1,34 @@
+# Resources for motion planning
+
+## Courses:
+### beginners:
```

Comment:
Capital B? Make similar changes where first letter is not capitalized (like books)